RetailCoder.VBE/App.cs

```diff
@@ -44,171 +34,27 @@ public sealed class App : IDisposable
 
         public App(IVBE vbe, 
             IMessageBox messageBox,
-            IParseCoordinator parser,
             IGeneralConfigService configService,
             IAppMenu appMenus,
             RubberduckCommandBar stateBar,
             IRubberduckHooks hooks,
             IVersionCheck version,
```

Comment:
@retailcoder  - `configService` can go pretty soon too. It's really only needed for localization and hotkeys - App is holding the dependency as a "proxy" for classes that actually need to respond to its events.  There's no reason that App should be calling methods directly on classes that can subscribe to an event handler.